packages/utils/src/abi_encoder/utils/signatureParser.ts

```diff
@@ -0,0 +1,101 @@
+import { DataItem } from 'ethereum-types';
```

Comment:
Why is this file name not camel-case? It should be.

Reply:
You mean snake-case right?

Reply:
You mean snake case?